### PR TITLE
Added missing 0x in addresses

### PR DESCRIPTION
**Pull request by 0xsarvesh**

### CODE REVIEW

**Comment by rahul7668gupta** [comment]
updated transfer tracker data:
Screenshot 2023-01-18 at 7 50 23 PM

updated token tracker data
Screenshot 2023-01-19 at 12 42 01 PM
Screenshot 2023-01-19 at 12 41 55 PM
Screenshot 2023-01-19 at 12 41 49 PM

**Comment by 0xsarvesh** [comment]
In second image no 0x in address & hash.

**Comment by rahul7668gupta** [comment]
updated code and pr screenshots

**Comment by rahul7668gupta** [comment]
this should be good now

File: transfer-tracker/src/helpers/utils.rs [hunk 1]
```diff
@@ -1,5 +1,3 @@
-use num_bigint::BigUint;
-
-pub fn bigint_from_bytes(value: Vec<u8>) -> BigUint {
-    BigUint::from_bytes_be(value.as_slice())
-}
\ No newline at end of file
+pub fn format_address(address: String) -> String {
```
Comment (0xsarvesh) [inline transfer-tracker/src/helpers/utils.rs]: Please rename this function in both the substreams

Comment (rahul7668gupta) [reply transfer-tracker/src/helpers/utils.rs]: done

**Pull request merged**